docs/source/running-jobs/many-jobs.rst

```diff
@@ -240,11 +240,77 @@ Scale Out to Many Jobs
 With the above setup, you can now scale out to run many jobs in parallel. You
 can either use SkyPilot CLI with many config files or use SkyPilot Python API.
 
-With CLI and Config Files
-~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can run many jobs in parallel by (1) creating multiple config files and (2)
-submitting them as :ref:`SkyPilot managed jobs <managed-jobs>`.
+Using Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To run many jobs at once, we will launch the jobs as :ref:`SkyPilot managed jobs <managed-jobs>`. We can control the hyperparameter environment variables independently for each managed job.
+
+You can use normal loops in bash or Python to iterate over possible hyperparamters:
+
+.. tab-set::
+
+    .. tab-item:: CLI
+        :sync: cli
+
+        .. code-block:: bash
+
+          job_idx=0
+          for lr in 0.01 0.03 0.1 0.3 1.0; do
+              for max_steps in 100 300 1000; do
+                  sky jobs launch -n train-job${job_idx} -y -d train-template.yaml \
+                    --env LR=${lr} --env MAX_STEPS=${max_steps} \
+                    --env WANDB_API_KEY # pick up from environment
+                  ((job_idx++))
```

Comment:
Add a comment saying: You can also write to an env file and use --env-file, as described above.

Reply:
I feel like it is okay to leave this out since the env-file example is right below this.